package.json

```diff
@@ -43,19 +21,40 @@
     "bench": "ocular-test bench",
     "bench-browser": "ocular-test bench-browser",
     "test": "ocular-test",
-    "test-browser": "ocular-test browser"
+    "test-browser": "ocular-test browser",
+    "test-size-explicit": "npm run build && webpack --config test/webpack.config.js --env.import-vec4-mat4-explicit",
+    "test-size": "npm run build && webpack --config test/webpack.config.js --env.import-nothing"
   },
   "dependencies": {
     "@babel/runtime": "^7.0.0",
     "gl-matrix": "^3.0.0"
   },
   "devDependencies": {
-    "@probe.gl/bench": "^3.0.0",
-    "@probe.gl/test-utils": "^3.0.0",
+    "@probe.gl/bench": "^3.0.2",
+    "@probe.gl/test-utils": "^3.0.2",
     "babel-plugin-transform-builtin-extend": "^1.1.2",
     "npx": "^10.2.0",
-    "ocular-dev-tools": "0.0.19",
+    "ocular-dev-tools": "0.0.24",
     "pre-commit": "^1.2.2",
-    "reify": "^0.18.1"
+    "reify": "^0.18.1",
+    "prettier": "^1.11.1",
+    "raw-loader": "^0.5.1",
+    "reify": "^0.18.1",
+    "source-map-loader": "^0.2.3",
+    "tap-browser-color": "^0.1.2",
+    "tape": "^4.5.1",
+    "tape-catch": "^1.0.4",
```

Comment:
prettier, source-map-loader, tape, tape-catch are dependencies of ocular-dev-tools